go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/crate-crypto/go-kzg-4844 v0.7.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0 // indirect
+	github.com/dustinxie/ecc v0.0.0-20210511000915-959544187564 // indirect
```

Comment:
The [ethereum golang crypto package](https://pkg.go.dev/github.com/ethereum/go-ethereum/crypto/secp256k1) provides an implementation for `secp256k1` signature algorithm. Our sdk already depends on the ethereum package as a [transitive dependency](https://github.com/project-alvarium/alvarium-sdk-go/blob/683fbd48cc7260df57217583ca09da29c3ac792e/go.mod#L23C2-L23C42) coming from the Hedera SDK. It might be worth checking the ethereum package out instead of adding a new dependency.

Reply:
> The [ethereum golang crypto package](https://pkg.go.dev/github.com/ethereum/go-ethereum/crypto/secp256k1) provides an implementation for `secp256k1` signature algorithm. Our sdk already depends on the ethereum package as a [transitive dependency](https://github.com/project-alvarium/alvarium-sdk-go/blob/683fbd48cc7260df57217583ca09da29c3ac792e/go.mod#L23C2-L23C42) coming from the Hedera SDK. It might be worth checking the ethereum package out instead of adding a new dependency.

I'm not sure about this given that the `dustinxie` pkg doesn't have any dependencies other than the Go stdlib. Remember the back and forth we had around the transitive deps when we brought the Hedera SDK in as a dependency? If we're ever able to decouple Hedera from the SDK (or if Hedera streamlines their dependencies), we should avoid coupling to `go-ethereum` somewhere else. If Jason can accomplish what he's looking to do with this cleaner pkg I'm not opposed to it.